script/approver.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+	"strings"
+)
+
+type Approver struct {
+	gitHub      GitHub
+	application Application
+}
+
+func (a *Approver) Approve() {
+	a.gitHub = GitHub{}
+	a.application = Application{}
+
+	if err := a.application.SetApprover(); err != nil {
+		a.printErrorAndExit(err)
+	}
+
+	if err := a.gitHub.Init(); err != nil {
+		a.printErrorAndExit(err)
+	}
+
+	if *a.gitHub.Issue.State == "closed" {
+		a.printErrorAndExit(fmt.Errorf("script run on closed issue"))
```

Comment:
This could be just an 
```suggestion
		a.printErrorAndExit(errors.New("script run on closed issue"))
```